postmaster/static/js/utils.js

```diff
@@ -16,25 +16,6 @@ function getUrlVars() {
 }
 
 
-// Inspired from https://github.com/janl/mustache.js/blob/master/mustache.js
-function filterText(text) {
```

Comment:
The `text` function escapes all text input so there's nowhere left in the code base that I am displaying user supplied text without having it go through the `text` function.
http://api.jquery.com/text/